test/test_bson_corpus.py

```diff
@@ -83,10 +84,12 @@
 to_bson_uuid_04 = functools.partial(encode,
                                     codec_options=codec_options_uuid_04)
 to_bson = functools.partial(encode, codec_options=codec_options)
-decode_bson = lambda bbytes: decode(bbytes, codec_options=codec_options)
+# decode_bson = lambda bbytes: decode(bbytes, codec_options=codec_options)
+decode_bson = lambda bbytes: decode(bbytes, codec_options=codec_options_no_tzaware)
```

Comment:
@ShaneHarvey it appears to me that we can either continue testing with or without `tz_aware`. In these changes, I asked @juliusgeo to move all testing to use tz-naive datetimes, but it can just as easily use tz-aware ones.

Reply:
I think it's fine to do either as long as we have testing for both True/False elsewhere. @juliusgeo can you confirm?
Can you also change this line to use functools.partial instead of a lambda?

Reply:
I am not sure where to look to verify that but I will respond to this comment later. I changed it to use functools.partial.